### PR TITLE
Fixing XSS Vulns in Fact Sources Modal

### PR DESCRIPTION
**Pull request by garunagiri**
## Description

The Fact Sources modal had 2 discovered XSS vulnerabilities. The changes in this PR sanitize XSS when information from the server is being placed into the web page.

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

I tested this manually to verify that the modal works as intended, and prevents XSS.

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: templates/sources.html [hunk 1]
```diff
@@ -183,21 +187,35 @@ <h3 id="relationship-name"></h3>
...
+                .append($("<p></p>")
+                    .text(value)
+                    .attr("contenteditable", "true")
+                )
+            )
+            .append($("<td></td>")
+                .append($("<p></p>")
+                    .html("&#x274C;")
+                    .attr("contenteditable", "true")
+                    .attr("onclick", "$(this).closest('tr').remove()")
```
Comment (b17zr) [inline templates/sources.html:215]: Could this use jQuery's `.click()` function? Something like (hoping this arrow function works right):

```
... .click(event => {
    $(event.currentTarget).closest('tr').remove();
});
```

File: templates/sources.html [hunk 2]
```diff
@@ -152,7 +152,11 @@ <h3 id="relationship-name"></h3>
                     }
                 });
                 if(!found){
-                    $('#profile-source-name').append('<option value="'+s.id+'">'+s.name+'</option>');
+                    $('#profile-source-name')
+                        .append($("<option></option>")
```
Comment (b17zr) [inline templates/sources.html:156]: At some point we will have a linter everywhere, but for now I think we're trying to match up with the Airbnb style guide. These double-quoted strings should be single-quoted: https://github.com/airbnb/javascript#strings--quotes

**Review by b17zr — CHANGES_REQUESTED** [review]
A couple small changes

**Review by protoli — APPROVED** [review]

File: templates/sources.html (hunk 2 above)
Comment (garunagiri) [reply templates/sources.html]: Changed my double quotes to singles

**Pull request merged**